### ISSUE TITLE
invalid device function

### ISSUE THREAD

**Issue by zyqz97** [Tong-ZHAO/Pixel2Mesh-Pytorch#16]
**There is some problem occur when i run the scripts.**

_Random Seed:  9909
training set 796
Start training!
[0: 0/796] train loss:  49.714088 
error in nnd updateOutput: invalid device function
error in nnd updateOutput: invalid device function
error in nnd updateOutput: invalid device function_

**Someone say that is because of the compute capablilty is not consistent with CUDA.
But i modify setup.py show below to match my GPU device:**

_from setuptools import setup
from torch.utils.cpp_extension import BuildExtension, CUDAExtension
cxx_args = ['-std=c++11']
nvcc_args = [
    '-gencode', 'arch=compute_52,code=sm_52', ]
setup(
    name='chamfer',
    ext_modules=[
        CUDAExtension('chamfer', [
            'chamfer_cuda.cpp',
            'chamfer.cu',
        ], extra_compile_args={'cxx': cxx_args, 'nvcc': nvcc_args}),
    ],
    cmdclass={
        'build_ext': BuildExtension
    })_

**It doesn't work. Could you help me?**
